gcp/modules/istio/hpa.tf

```diff
@@ -0,0 +1,119 @@
+resource "kubernetes_horizontal_pod_autoscaler" "istio-egressgateway" {
+  metadata = {
+    name      = "istio-egressgateway"
+    namespace = "istio-system"
+
+    labels = {
+      k8s-app = "istio"
+    }
+  }
+
+  spec {
+    max_replicas = 5
+    min_replicas = 2
+
+    scale_target_ref {
+      kind        = "Deployment"
+      name        = "istio-egressgateway"
+      api_version = "apps/v1beta1"
+    }
+
+    target_cpu_utilization_percentage = 80
```

Comment:
How did you determine this number?

Reply:
This is an existing configuration by Google - ie. it does not change anything, but has to be here, otherwise TF would try to remove it from the rule.